src/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+test('renders correctly', () => {
+  const tree = renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
Nice job using jest snapshots to test your components✨. However, according to the project requirements, you are also supposed to write unit tests for **all** your components using the react testing library. The react testing library provides many useful functions for interacting with your components during testing. Take a look at this [tutorial](https://testing-library.com/docs/react-testing-library/example-intro) to see examples of the library in action. You can also have a look at this [cheatsheet](https://testing-library.com/docs/react-testing-library/cheatsheet) to see common queries used in the library. 

Please add more tests for your components using functions from the react testing library🙏